src/RockBridge.cpp

```diff
@@ -135,30 +145,53 @@ void RockBridge::updateBegin(common::UpdateInfo const& info)
     }
 }
 
-void RockBridge::updateEnd()
-{
-}
-
-RockBridge::RockBridge()
-{
-}
 
-RockBridge::~RockBridge()
+void RockBridge::instantiatePluginComponents(sdf::ElementPtr modelElement, ModelPtr model)
 {
-    // Delete pointers to activity
-    for(Activities::iterator activity_it = activities.begin(); 
-            activity_it != activities.end(); ++activity_it)
+    if(modelElement->HasElement("plugin"))
```

Comment:
We found out with Gustavo that GetElement returns a NULL pointer if there are no elements, so the outermost if() is not needed.